packages/sdk/src/block/block.js

```diff
@@ -0,0 +1,26 @@
+import {send} from "../send/send.js"
+import {getBlock} from "../build/build-get-block"
+import {decodeResponse as decode} from "../decode/decode.js"
+import {invariant} from "@onflow/util-invariant"
+import {atBlockHeight, atBlockId} from "../sdk"
+
+export function block({sealed = false, id, height} = {}) {
+  invariant(
+    !((sealed && id) || (sealed && height)),
+    `Cannot pass "sealed" with "id" or "height"`
```

Comment:
If we prepend an identifier for the warning message in the invariant message it can be helpful when debugging.

Something like `fcl.block --  Cannot pass "sealed" with "id" or "height"`

Reply:
@JeffreyDoyle Good idea. Added but made it `method: block` since this can be accessed from `sdk` or `fcl`.